.settings/language.settings.xml

```diff
@@ -1,9 +1,28 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <project>
-	<configuration id="cdt.managedbuild.toolchain.gnu.base.1461790692" name="Default">
-		<extension point="org.eclipse.cdt.core.LanguageSettingsProvider">
-			<provider copy-of="extension" id="org.eclipse.cdt.ui.UserLanguageSettingsProvider"/>
-			<provider-reference id="org.eclipse.cdt.managedbuilder.core.MBSLanguageSettingsProvider" ref="shared-provider"/>
-		</extension>
-	</configuration>
+    	
```

Comment:
Actually @dddejan, since you are an eclipse user, it would be great to have your input for how valuable it is to have those files under version control. [There seems to be some disagreement about that.](https://stackoverflow.com/questions/1551008/why-does-eclipse-create-a-settings-directory?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa)